Felienne/hedy
# [BUG FIX] Correctly highlight code snippet example on customize adventure page

**Description**
In this PR we fix a small issue where the syntax highlighting on the customize adventure page didn't work due to still being dependent on the `id` attribute rather than the currently used `level` attribute. We als make the code snippet itself language dependent.

**Fixes**
This PR fixes #2713.

**How to test**
Make sure you are a teacher. Create a new adventure and verify that the "hello world" snippet is correctly displayed with the syntax highlighting.
